ReadMe.pod

```diff
@@ -10,8 +10,7 @@ See http://github.com/ingydotnet/swim-pm#readme
 
 Bash+(1) - Modern Bash Programming
 
-=for html
-<a href="https://travis-ci.org/ingydotnet/bashplus"><img src="https://travis-ci.org/ingydotnet/bashplus.png" alt="bashplus"></a>
```

Comment:
This broke because you don't have `Swim::Plugin::badge` installed.
Run `cpanm Swim::Plugin::badge` to install that.